src/gpu/intel/gemm/jit/include/internal/utils.hpp

```diff
@@ -21,9 +21,12 @@
 #include <string>
 #include <sstream>
 
+#if (defined(_MSVC_LANG) && (_MSVC_LANG >= 202002L))
+ || (defined(__cplusplus) && (__cplusplus >= 202002L))
```

Comment:
I actually see this check further down in the file when `std::source_location` is used:

```c++
#if defined(__cpp_lib_source_location) && __cpp_lib_source_location >= 201907L
```

BTW, I think `__cpp_lib_source_location ` based check should work for the header as well.

Can we put it for the header as well to have these two places aligned?